specification/monitor/resource-manager/microsoft.insights/2017-11-01-preview/baseline_API.json

```diff
@@ -0,0 +1,347 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "title": "MonitorManagementClient",
+      "x-ms-code-generation-settings": {
+        "name": "MonitorManagementClient"
+      },
+      "version": "2017-11-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+      "https"
+    ],
+    "consumes": [
+      "application/json"
+    ],
+    "produces": [
+      "application/json"
+    ],
+    "security": [
+      {
+        "azure_auth": [
+          "user_impersonation"
+        ]
+      }
+    ],
+    "securityDefinitions": {
+      "azure_auth": {
+        "type": "oauth2",
+        "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+        "flow": "implicit",
+        "description": "Azure Active Directory OAuth2 Flow",
+        "scopes": {
+          "user_impersonation": "impersonate your user account"
+        }
+      }
+    },
+    "paths": {
+      "/{resourceUri}/providers/microsoft.insights/baseline/{metricName}": {
```

Comment:
Description should state/show examples of resourceUri

Reply:
Added the description to the ResourceUriParameter parameter part.